### ISSUE TITLE
VoiceOver doesn't read toolbar option labels in Chrome

### ISSUE THREAD

**Issue by marjon4** [ckeditor/ckeditor5#10687]
## 📝 Provide detailed reproduction steps (if any)

1. Activate VoiceOver on Mac (command + F5)
2. Navigate to CKEditor text editor by tab
3. Navigate to toolbar options (option + F10)

### ✔️ Expected result

VoiceOver should read "bold, button", if focus is on the bold formatting option in the toolbar.

(There seems to be a similar problem with JAWS on Chrome/Windows too. In VoiceOver on Safari/Mac it works as expected though.)

### ❌ Actual result

VoiceOver only reads "button", which leaves screen reader users unaware of what the button does.

### ❓ Possible solution

Did just give it a quick try and put aria-label="bold" directly on the button-element instead of an aria-labelledby reference like the current solution. Worked both in Chrome and Safari.

## 📃 Other details

* Browser: Chrome
* OS: MacOS Big Sur
* First affected CKEditor version: 28.0.0
* Installed CKEditor plugins:  Bold, Italic, Paragraph, PasteFromOffice

**Comment by JanettHolst290490** [comment]
This is also an issue in Voiceover for Mac.
If you navigate by form elements (VO + command + J) or if you navigate by listing form elements (Control + option + u)

**Comment by bbenjamin** [comment]
I'm unable to reproduce the reported issue, here's a video navigating with Chrome + Voiceover https://youtu.be/s1u_Z6n6lZw
Navigating by form element works for me as well
![voiceover-chrome-rotor](https://user-images.githubusercontent.com/1742345/169888682-d1ae8a27-7c4f-4636-93b3-b60ee929ec22.jpg)
